test/ui/starter-select.test.ts

```diff
@@ -495,11 +495,11 @@ describe("UI - Starter select", () => {
     // expect(starterSelectUiHandler.cursorObj.y).toBe(10);
 
     game.onNextPrompt("SelectStarterPhase", UiMode.CONFIRM, () => {
-      const handler = game.scene.ui.getHandler() as StarterSelectUiHandler;
+      const handler = game.scene.ui.getCurrentHandler() as StarterSelectUiHandler;
```

Comment:
NIT: Use generic param `<T>`

```suggestion
      const handler = game.scene.ui.getCurrentHandler<StarterSelectUiHandler>();
```